ftw/contentpage/tests/pages.py

```diff
@@ -1,45 +1,48 @@
-from ftw.testing import browser
-from ftw.testing.pages import Plone
+from plone import api
 
 INDEX_XPATH = '//*[contains(concat(" ", normalize-space(@class), " "), ' + \
     '" letter-index ")]'
 
 
-class SubjectListingView(Plone):
+class SubjectListingView(object):
+    
+    def __init__(self, browser):
```

Comment:
You could simplify this by using a default browser:

```python
from ftw.testbrowser import browser as default_browser

...
    def __init__(self, browser=default_browser):
       ...
```

Compare with https://github.com/4teamwork/ftw.testbrowser/blob/master/ftw/testbrowser/pages/plone.py#L5

----

Background:
Although the testbrowser usually is used in a singleton mannor and can be imported (`from ftw.testbrowser import browser`) it is possible to use multiple browser instances in parallel (e.g. in a multi user situation). By allowing the browser to be passed in we support multiple browser in the page objects too.